Assets/Scripts/Spawning/ResourceSpawning/ResourceSpawner.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Examen.Spawning.ResourceSpawning
+{
+    public class ResourceSpawner : MonoBehaviour
+    {
+        [SerializeField] private List<SpawnAreas> spawnAreas;
+
+        /// <summary>
+        /// Spawns randomly resources
+        /// </summary>
+        public void InitializedSpawning()
+        {
+            for (int i = 0; i < spawnAreas.Count; i++)
+            {
+                SpawnAreas area = spawnAreas[i];
+                float spawnPercentage = 100;
```

Comment:
Should this be hard coded? if so make it a constant otherwise make it a variable.

Reply:
Yeah because it is precentage, Maxspawnpercentage is always 100%